packages/xo-web/src/xo-app/vm/action-bar.js

```diff
@@ -51,6 +62,14 @@ const vmActionBarByState = {
           pending={includes(vm.current_operations, 'copy')}
         />
       )}
+      {!isSelfUser && canAdministrate && (
+        <Action
+          handler={cloneToTemplate}
+          icon='template'
+          label={_('cloneAsTemplate')}
+          pending={includes(vm.current_operations, 'clone_to_template')}
+        />
+      )}
```

Comment:
I don't think this should go in the action bar since it's not something that users want to do very often.